eng/pipelines/cadl-ranch-typespec.yml

```diff
@@ -4,22 +4,23 @@ trigger:
       - main
   paths:
     include:
-      - packages/typespec-go/test/cadlranch
+      - packages/typespec-go/test/http-spec
```

Comment:
can we rename this file? I think we shall not sue cadl-ranch in our repos in the future.

Reply:
rename to `spector-typespec.yml`